Detections/SymantecProxySG/UserAccessedSuspiciousURLCategories.yaml

```diff
@@ -0,0 +1,22 @@
+id: fb0f4a93-d8ad-4b54-9931-85bdb7550f90
+name: User Accessed Suspicious URL Categories
+description: |
+  'Creates an incident in the event the requested URL accessed by the user has been identified as Suspicious, Phishing, or Hacking.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - SymantecProxySG
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  let timeframe = ago(1h);
+  SymantecProxySG
+  | where TimeGenerated >= timeframe
+  | mv-expand cs_categories
+  | where cs_categories has_any ("Suspicious","phishing", "hacking")
+  | summarize count() by sc_filter_result, cs_userdn, c_ip, cs_host, Computer,  tostring(cs_categories)
```

Comment:
Bring thru Time - | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by sc_filter_result, cs_userdn, c_ip, cs_host, Computer,  tostring(cs_categories)

Reply:
Corrections made and pushed